blueocean-pipeline-api-impl/pom.xml

```diff
@@ -74,6 +74,11 @@
             <artifactId>pipeline-graph-analysis</artifactId>
             <version>1.1</version>
         </dependency>
+        <dependency>
```

Comment:
I am not sure what the plan is, but I don't think longer term we want any dependencies from pipeline-api-impl to any specific SCM provider right @i386 - ie this should be provided by an external plugin using an extension point I think? (right @i386)